src/app/pages/home/latest-comments/latest-comments.component.ts

```diff
@@ -0,0 +1,47 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core';
+import { DataError } from 'src/app/types/data-error';
+import { LatestCommentsGQL, LatestCommentsQuery } from 'src/generated/graphql';
+import { LoadingSpinnerService } from '../loading-spinner.service';
+
+@Component({
+  selector: 'app-latest-comments',
+  templateUrl: './latest-comments.component.html',
+  styleUrls: ['./latest-comments.component.scss'],
+})
+export class LatestCommentsComponent implements OnInit {
+  @Output() errorEvent = new EventEmitter<DataError>();
+
+  loading: boolean;
+  comments: LatestCommentsQuery['latestComments']['items'];
+
+  constructor(
+    private latestCommentsGQL: LatestCommentsGQL,
+    private loadingSpinnerService: LoadingSpinnerService
+  ) {}
+
+  ngOnInit(): void {
+    this.loading = true;
+    this.loadingSpinnerService.pushLoader();
+
+    const latestComments$ = this.latestCommentsGQL.fetch({
+      input: {
+        pageSize: 5,
+        pageNumber: 1,
+      },
+    });
+
```

Comment:
Latest comments should be piped/switchMapped with authService.currentUser stream, so that when user logs out/logs in comments are refetched...
might want to use latestDifficultyVotes or latestAscents as an example

Reply:
That makes sense. I added these changes.